src/s2/encoded_s2point_vector.cc

```diff
@@ -124,7 +137,7 @@ bool EncodedS2PointVector::Init(Decoder* decoder) {
 vector<S2Point> EncodedS2PointVector::Decode() const {
   vector<S2Point> points;
   points.reserve(size_);
-  for (int i = 0; i < size_; ++i) {
+  for (size_t i = 0; i < size_; ++i) {
     points.push_back((*this)[i]);
```

Comment:
In general instead of warning about compare signed and unsigned, now we have warning about implicit conversation big unsigned to small signed (almost all `operator[]` works with int)

Am I right?

But I also think it's not real important, because after system include PR users can disable it

Reply:
>Am I right?

No, at least not with the default g++ warnings.

We will be moving to more use of unsigned types in the future.

Reply:
Ok